### ISSUE TITLE
No helpers.sh file?

### ISSUE THREAD

**Issue by frankshearar** [hpi-swa/smalltalkCI#24]
I'm sure it's just me being dumb. https://travis-ci.org/frankshearar/Control/jobs/99882794 fails because the above-mentioned file doesn't exist:

````
Downloading and extracting smalltalkCI
0.09s$ wget -q -O smalltalkCI.zip https://github.com/hpi-swa/smalltalkCI/archive/master.zip
0.03s$ unzip -q -o smalltalkCI.zip
0.00s0.01s$ source env_vars
0.00s
0.00s$ $SMALLTALK_CI_HOME/run.sh
/home/travis/smalltalkCI-master/run.sh: line 5: helpers.sh: No such file or directory
````
 The .travis.yml file (https://github.com/frankshearar/Control/blob/update-to-new-travis-abilities/.travis.yml) looks OK to my untrained eye.

What have I messed up?

**Comment by fniephaus** [comment]
Looks like I merged #22 too fast, it was a bug in smalltalkCI.
Could you please restart the builds and check if the problem is resolved?
Thanks!

**Comment by fniephaus** [comment]
I just forked your repository to run the tests myself and then I realized that you are using a Metacello configuration instead of a baseline. I will add support for configurations as soon as I can.
